api/db-cdr/csv/achilles_analysis.csv

```diff
@@ -10,18 +10,25 @@ analysis_id,"analysis_name","stratum_1_name","stratum_2_name","stratum_3_name","
 12,"Number of persons by race and ethnicity","race_concept_id","ethnicity_concept_id","","","","pie","counts"
 603,"Number of distinct procedure occurrence concepts per person","","","","","","column","counts"
 606,"Age distribution by procedure_concept_id","procedure_concept_id","gender_concept_id","","","","box","distribution"
-1815,"Distribution of numeric values, by measurement_concept_id and unit_concept_id","","","","","","column","counts"
+1806,"Distribution of age by measurement_concept_id","measurement_concept_id","gender_concept_id","null","null","null","box","distribution"
+1815,"Distribution of numeric values, by measurement_concept_id and unit_concept_id","measurement_concept_id","unit_concept_id","","","","box","distribution"
 1816,"Distribution of low range, by measurement_concept_id and unit_concept_id","","","","","","column","counts"
 1817,"Distribution of high range, by observation_concept_id and unit_concept_id","","","","","","column","counts"
+1900,"Measurement response gender distribution","measurement_concept_id","gender_concept_id","","value","unit_source_value","column","counts"
+1901,"Measurement response age distribution","measurement_concept_id","age_decile","","value","unit_source_value","column","counts"
+1911,"Measurement Male Distribution","measurement_concept_id","gender_concept_id",NULL,"value","unit_source_value","column","counts"
+1912,"Measurement Female Distribution","measurement_concept_id","gender_concept_id",NULL,"value","unit_source_value","column","counts"
 2000,"Number of patients with at least 1 Dx and 1 Rx","","","","","","column","counts"
 2001,"Number of patients with at least 1 Dx and 1 Proc","","","","","","column","counts"
 2002,"Number of patients with at least 1 Meas, 1 Dx and 1 Rx","","","","","","column","counts"
 2003,"Number of patients with at least 1 Visit","","","","","","column","counts"
-3000,"Participant Count","concept_id",NULL,"domain_id",NULL,NULL,"column","counts"
+3000,"Participant Count","concept_id","bin_size","domain_id","bin_min","bin_max","column","counts"
 3004,"Number of persons with at least one concept by concept_id by calendar year by gender by age decile","concept_id","calendar year","gender_concept_id","age decile","","column","counts"
 3006,"Age distribution by gender","concept_id","gender_concept_id","","","","box","distribution"
 3101,"Gender","concept_id","ppi_sex_at_birth_concept_id","domain_id",NULL,NULL,"pie","counts"
 3102,"Age","concept_id","age_decile","domain_id","null","","column","counts"
+3103,"Race","survey_concept_id","race_concept_id","domain_id",NULL,NULL,"column","counts"
```

Comment:
This should just be "concept_id" not survey_concept_id , in case we do this for more concepts

Reply:
Addressed